doc/README.md

```diff
@@ -0,0 +1,23 @@
+# Ceph - a scalable distributed storage system
```

Comment:
the major part of this document is literally copied from https://github.com/ceph/ceph/blob/master/README.md . how are we going to structure them without replicating the contents?

Reply:
You are right. But is it a problem?

The info I included here is completely relevant to the material available in this directory.

The fact that this info is also available in another place shouldn't prevent it to be available exactly where it is most relevant, should it?

Maybe, for the sake of avoiding repetition, we should keep this info here and in https://github.com/ceph/ceph/blob/master/README.md we would just point here for more info building the documentation. Does it seems reasonable?